src/modules/trading/components/trading--form/trading--form.styles.less

```diff
@@ -136,24 +166,24 @@
   border: 1px solid @color-dark-gray;
   display: flex;
   flex-direction: row;
-  font-family: 'Roboto Mono', monospace;
+  font-family: "Roboto Mono", monospace;
 
   > input[type="number"] {
     background-color: @color-dark-grayish-purple;
     border-width: 0;
     color: @color-white;
-    font-family: 'Roboto Mono', monospace;
+    font-family: "Roboto Mono", monospace;
     font-size: @font-size-medium;
 
     &::placeholder {
-      color: rgba(255, 255, 255, 0.2);
-      font-family: 'Roboto Mono', monospace;
+      color: fade(@color-secondary-text, 50%);
+      font-family: "Roboto Mono", monospace;
       font-size: @font-size-medium;
     }
   }
 
   &.error {
-    border: 0.0625rem solid rgba(255, 39, 39, 0.5) !important;
+    border: 0.0625rem solid fade(@color-error-text, 50%) !important;
```

Comment:
should avoid or get rid of these !important things if possible

Reply:
removed the importants.